src/neo-vm/OpCode.cs

```diff
@@ -443,6 +443,10 @@ public enum OpCode : byte
         /// A map is taken from top of the main stack. The values of this map are put on top of the main stack.
         /// </summary>
         VALUES = 0xCD,
+        /// <summary>
+        /// A struct is removed from top of the main stack. Its elements are put on top of the main stack (in reverse order) and the struct size is also put on main stack.
```

Comment:
I guess this comment is reversed.... The purpuse is to extract elements from the main stack and put a struct, or am I mistaken?